src/AlphaRPC/Console/Command/GenerateSupervisorConfig.php

```diff
@@ -78,9 +78,9 @@ public function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $group = $input->getArgument('group');
-        if (!preg_match('/^[A-Za-z0-9]+$/', $group)) {
+        if (!preg_match('/^[A-Za-z0-9]+(-[A-Za-z0-9]+)*$/', $group)) {
```

Comment:
Why the extra match? The regex `'/^[A-Za-z0-9\-]+$/'` is much cleaner.

Reply:
Using  '/^[\w-]+$/' would also allow for underscore to be used.

Please use the suggested regex or if you insist on not starting with a dash ^[A-Za-z0-9]+[\-A-Za-z0-9]*$

Reply:
I propose to remove the group name validation because it is not the responsibility of AlphaRPC. If the user uses invalid characters then they will find out because supervisor will not start. 

It is also incorrect because it looks like the only disallowed characters for group names in the supervisor configuration are " ", ":" and brackets. 
https://github.com/Supervisor/supervisor/issues/647
https://github.com/Supervisor/supervisor/blob/3.1.3/supervisor/datatypes.py#L16-L22

Reply:
I still think in general its not bad thing to validate something as early as possible.
But since this group name can contain almost any character i have to agree that the validation can be removed.